### PR TITLE
Add Run / Configure instructions to README

### PR DESCRIPTION
**Pull request by Kache**
And fix whitespace

Fixes: #613

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -94,9 +95,17 @@ brew tap homebrew/cask-versions
 brew install --cask background-music-pre
 ```
 
+# Configure
+
+Select **Background Music** for `Output` and `Input` under `System Preferences > Sound`.
```
Comment (kyleneideck) [inline README.md:100]: Background Music should automatically set your default output device to the Background Music device when it starts up. Is that not working for you? It's also supposed to change your default device back when you close `Background Music.app`.

You should only need your default output device (and system output, i.e. sound effects) set to the Background Music device, though, not your default input device.

What do you think about moving this to the troubleshooting section? Maybe like this:

```
Make sure you have **Background Music** selected for `Output` in `System Preferences > Sound`.
```

File: README.md [hunk 2]
```diff
@@ -94,9 +95,17 @@ brew tap homebrew/cask-versions
...
+
+### Launch at Startup
+
+Add **Background Music** to `System Preferences > Users & Groups > Current User > Login Items`.
```
Comment (kyleneideck) [inline README.md:104]: Thanks, I think adding this is a really good idea. One nitpick: we should probably make it clear that this step is optional. Maybe something like this.

```suggestion
If you'd like Background Music to run when you start your computer, add **Background Music** to `System Preferences > Users & Groups > Current User > Login Items`.
```

File: README.md (hunk 1 above)
Comment (Kache) [reply README.md]: Updated PR to reflect this behavior. This is a minor thing -- I was just momentarily confused after install b/c I didn't know where the executable was added, if I had to add something to my `$PATH`, or if it installed as a driver thing and it'd already "activated". For example, some `brew` formulas print out "how to get it running" step of "add <THIS> to your ~/.zshrc" after install, as well as in `brew info`.

**Review by kyleneideck — APPROVED** [review]

**Comment by kyleneideck** [comment]
Looks great. Merged, thanks.

**Pull request merged**